grasp_planning_server:
    ros__parameters:
        object_mesh_path: '/ros2_ws/src/wros2_tutorials/meshes/tubebig.stl'
        obstcl_mesh_path: '/ros2_ws/src/wros2_tutorials/meshes/table.stl'
        vis_failures: True
        save_results: False
        gripper_name: 'robotiqhe'
        antipodal_grasp:
            angle_between_contact_normals: 90
            openning_direction: 'loc_x' 
            max_samples: 4
            min_dist_between_sampled_contact_points: .016
            contact_offset: .016